compute_rouge_scores.py

```diff
@@ -9,41 +9,48 @@
     gold_sums, gold_metadata = get_gold_xsum_data()
 
     MODEL_RESULTS = {
-        # "BartGEF": load_summaries_from_logs(
-        #     "results/fbs-logs/bart-full-classifier-knnv1.json", max_iterations=5
-        # ),
+        "BartGEF": load_summaries_from_logs(
+            "results/fbs-logs/bart-full-classifier-knnv1.json", max_iterations=5
+        ),
         "PegasusGEF": load_summaries_from_logs(
             "results/fbs-logs/pegasus-full-classifier-knnv1.json", max_iterations=5
         ),
     }
 
     sumtool_models = [
-        # ("facebook-bart-large-xsum", "BartBaseline"),
-        # ("meng-rl", "MengRL"),  # Hallucinated, but factual! Paper
-        # ("pinocchio", "Pinocchio"),  # King et. al paper
-        # ("chen-corrector", "Corrector"),  # Chen. et al replication project
-        # ("google-pegasus-xsum", "PegasusBaseline"),
+        ("facebook-bart-large-xsum", "BartBaseline"),
+        ("meng-rl", "MengRL"),  # Hallucinated, but factual! Paper
+        ("pinocchio-fallback-ours", "Pinocchio"),  # King et. al paper
+        ("pinocchio-fallback-theirs", "PinocchioTheirFallback"),  # King et. al paper
+        ("pinocchio-with-skip", "PinocchioSkip"),  # King et. al paper
+        ("chen-corrector", "Corrector"),  # Chen. et al replication project
+        ("google-pegasus-xsum", "PegasusBaseline"),
     ]
     for (sumtool_name, model_label) in sumtool_models:
         dataset = get_summaries("xsum", sumtool_name)
         MODEL_RESULTS[model_label] = (
             {sum_id: x["summary"] for sum_id, x in dataset.items()},
             {},
-            {}
+            {},
         )
 
     rouge_scores_by_model = {}
 
-    for model_label, (sums_by_id, sum_ents_by_id, failed_sums_by_id) in MODEL_RESULTS.items():
+    for model_label, (sums_by_id, sum_ents_by_id, failed_sums_by_id) in tqdm(
+        list(MODEL_RESULTS.items())
+    ):
         rouge1 = []
         rouge2 = []
         rougeL = []
-        for sum_id, summary in tqdm(list(sums_by_id.items())):
-            reference = gold_sums[sum_id]
```

Comment:
Silly bug was leading to incorrect rouge scores. This should be:
```
reference = gold_sums[sum_id]["summary"]
```

The dict was being coerced to a string with extra symbols which led to a drop.